### PR TITLE
[WIP] asset catalog suggestion

### PR DESCRIPTION
**Pull request by mfrawley**
Related to #7 Support for asset catalogs

### CODE REVIEW

File: DBChooser/UI/DBCStyledErrorView.m [hunk 1]
```diff
@@ -130,7 +130,7 @@ - (void)layoutSubviews {
     CGFloat postTitleViewPadding = isPortrait ? [self padding2PortraitOverride] : [self padding2LandscapeOverride];
     CGFloat postSubtitleViewPadding = isPortrait ? [self padding3PortraitOverride] : [self padding3LandscapeOverride];
     CGFloat bottomPadding = isPortrait ? [self bottomPaddingPortraitOverride] : [self bottomPaddingLandscapeOverride];
-    UIEdgeInsets contentInsets = UIEdgeInsetsMake(edgeInsets.top + isPortrait ? [self topPaddingPortraitOverride] : [self topPaddingLandscapeOverride], 0, edgeInsets.bottom + bottomPadding, 0);
+    UIEdgeInsets contentInsets = UIEdgeInsetsMake((edgeInsets.top + isPortrait) ? [self topPaddingPortraitOverride] : [self topPaddingLandscapeOverride], 0, edgeInsets.bottom + bottomPadding, 0);
```
Comment (mfrawley) [inline DBChooser/UI/DBCStyledErrorView.m:133]: Fixed operator precedence warning here

File: DBChooser/UI/DBAppearance.m [hunk 2]
```diff
@@ -22,42 +22,54 @@ + (void)customizeNavBarForContainer:(Class<UIAppearanceContainer>)cls {
...
+    UIImage *blueBackLandscapePressed = [DBAppearance loadImageNamed:@"blue_back_button_landscape_pressed" fromBundle:bundle];
+    
     [[UINavigationBar appearanceWhenContainedIn:cls, nil]
-     setBackgroundImage:[[UIImage imageNamed:@"DBChooser.bundle/header"] resizableImageWithCapInsets:navInsets]
+     setBackgroundImage:[header resizableImageWithCapInsets:navInsets]
      forBarMetrics:UIBarMetricsDefault];
     UIEdgeInsets button_insets = UIEdgeInsetsMake(BarButtonCapHeight, BarButtonCapWidth, BarButtonCapHeight, BarButtonCapWidth);
     [[UIBarButtonItem appearanceWhenContainedIn:[UINavigationBar class], cls, nil]
-     setBackgroundImage:[UIImage imageNamed:@"DBChooser.bundle/blue_button"]
+      setBackgroundImage:blueButton
```
Comment (varenc) [inline DBChooser/UI/DBAppearance.m]: nit: looks like you inserted a leading space here accidentally :-)

Comment (varenc) [inline DBChooser/UI/DBAppearance.m:41]: nit: looks like you inserted a leading space here accidentally

**Comment by varenc** [comment]
thanks for this! This code could definitely use the love.

I'm happy to merge this in.  Though how do people typically deal with version references in podspecs? Is there way to centralize the version information somehow? I don't like all the places I'll have to change it when we do another release.

**Comment by mfrawley** [comment]
@varenc I wouldn't merge it yet, I haven't tested it all yet, I'll mark the PR as WIP until it's at least in a "works for me" state. As regards the version, if you include a podspec here and use a ruby podspec it can do anything it likes to read the version from elsewhere. Typically I do just manually tag the repo in question and bump the version in the podspec. Alternatively you could generate the json podspec from a template..

**Pull request closed without merging**